profiles/orcid.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+import requests
+from requests import Response
+
+API_VERSION = 'v2.0'
+
+
+class OrcidClient(object):
+    def __init__(self, api_uri: str) -> None:
+        self.api_uri = api_uri
+
+    def get_record(self, orcid: str, access_token: str) -> dict:
+        response = self._get_request('{}/record'.format(orcid), access_token)
+
+        return json.loads(response.text)
+
+    def _get_request(self, path, access_token) -> Response:
+        uri = '{}/{}/{}'.format(self.api_uri, API_VERSION, path)
+        headers = {'Accept': 'application/orcid+json', 'Authorization': 'Bearer ' + access_token}
+
+        response = requests.get(uri, headers=headers)
+        response.raise_for_status()
```

Comment:
this would be a good place for explicit logging

Reply:
All responses as `DEBUG`?

Reply:
since for now it's only `GET`, `DEBUG`; if we had `POST` or similar, would go for an higher level

Reply:
`requests` logs HTTP requests/responses anyway, maybe better to log messages like 'Failed to load record'.

Reply:
have yet to see a comprehensive log from `requests` and underlying library, let's wait

Reply:
Added https://github.com/elifesciences/profiles/pull/26/commits/01f692e4cbefcad5d33b0b36ea19afda7c8a2c8f.